deb/brave-keyring/DEBIAN/postinst

```diff
@@ -2,3 +2,9 @@
 
 # Delete (expired) key added by old install instructions
 apt-key --keyring /etc/apt/trusted.gpg del D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
```

Comment:
It might be time to retire this old cleanup line. It's causing a warning about `apt-key` being deprecated I believe.